### PR TITLE
feat: allow specifiying the number of print copies

### PR DESCRIPTION
**Pull request by eventualbuddha**
Refs votingworks/vxmail#147

### CODE REVIEW

File: src/ipc/print.ts [hunk 1]
```diff
@@ -43,6 +49,10 @@ async function printData({
     lprOptions.push('InputSlot=' + paperSource)
   }
 
+  if (typeof copies !== 'undefined') {
+    lprOptions.push('-#', copies.toString())
```
Comment (benadida) [inline src/ipc/print.ts:53]: should we check that copies is an integer (as opposed to having fractional copies)?

File: src/ipc/print.ts [hunk 2]
```diff
@@ -57,9 +67,29 @@ export default function register(ipcMain: IpcMain): void {
...
+      if (
+        typeof paperSource !== 'undefined' &&
+        typeof paperSource !== 'string'
+      ) {
+        throw new TypeError(
+          `paperSource expected to be a string, got: ${inspect(paperSource)}`,
+        )
+      }
+
+      if (typeof copies !== 'undefined' && typeof copies !== 'number') {
```
Comment (benadida) [inline src/ipc/print.ts:87]: probably here for the integer check.

File: src/preload.ts [hunk 3]
```diff
@@ -25,34 +25,33 @@ import FileWriter from './utils/FileWriter'
...
+    deviceName?: string,
+    paperSource?: string,
+    copies?: number,
+  ): Promise<void>
+  public async print(
+    deviceNameOrOptions?: KioskBrowser.PrintOptions | string,
+    paperSource?: string,
+    copies?: number,
+  ): Promise<void> {
+    const options = (typeof deviceNameOrOptions === 'string'
```
Comment (benadida) [inline src/preload.ts:39]: nice backwards compatibility

File: src/ipc/print.ts (hunk 2 above)
Comment (eventualbuddha) [reply src/ipc/print.ts]: Done. Replaced the manual checks with a Zod schema.

**Pull request merged**